lib/ansible/modules/cloud/amazon/cloudfront_info.py

```diff
@@ -154,23 +156,38 @@
 # Note: These examples do not set authentication details, see the AWS Guide for details.
 
 # Get a summary of distributions
-- cloudfront_facts:
+- cloudfront_info:
     summary: true
+  register: result
 
 # Get information about a distribution
-- cloudfront_facts:
+- cloudfront_info:
     distribution: true
     distribution_id: my-cloudfront-distribution-id
+  register: result_did
+- debug:
+    msg: "{{ result_did['cloudfront']['my-cloudfront-distribution-id'] }}"
 
 # Get information about a distribution using the CNAME of the cloudfront distribution.
-- cloudfront_facts:
+- cloudfront_info:
     distribution: true
     domain_name_alias: www.my-website.com
+  register: result_website
+- debug:
+    msg: "{{ result_website['cloudfront']['www.my-website.com'] }}"
 
-# Facts are published in ansible_facts['cloudfront'][<distribution_name>]
+# When the module is called as cloudfront_facts, return values are published
+# in ansible_facts['cloudfront'][<id>] and can be used as follows.
+# Note that this is deprecated and will stop working in Ansible 2.13.
+- cloudfront_facts:
+    distribution: true
+    distribution_id: my-cloudfront-distribution-id
```

Comment:
This seems to be the only place we're documenting this for posterity - I'd be ok with just removing the `cloudfront_facts` info unless you want to do the same for all the other modules?

Reply:
Right now, I've only added this part to modules which were documenting how to use the facts before. The other modules didn't do that.